ro-base.json

```diff
@@ -854,10 +854,9 @@
       }
     }, {
       "id" : "http://purl.obolibrary.org/obo/RO_0001018",
-      "lbl" : "contained in",
+      "lbl" : "obsolete contained in",
       "type" : "PROPERTY",
       "meta" : {
-        "subsets" : [ "http://purl.obolibrary.org/obo/ro/subsets#ro-eco" ],
         "basicPropertyValues" : [ {
```

Comment:
obsolete rels cant have logical axioms. Why didn't QC say anything?

Reply:
No QC is in place for this right now.

Reply:
No problem, lets defer.